cases/test_boards_page.py

```diff
@@ -37,25 +35,26 @@ def tearDown(self):
         self.driver.quit()
 
     def test_board_create_success(self):
-        self.boards_page.create_board(self.BOARD_NAME)
+        board_name = f'test_board_create_success{str(time.time())}'
```

Comment:
may be?
cases/test_join_page.py:34:        login = '{}{}'.format(Faker().first_name(), Faker().random_int())[:15]